### DISCUSSION TITLE
"500 Internal Server Error" when attempting to accept GitHub Classroom assignment

### DISCUSSION THREAD

**Discussion by benjamingadd** [community/community#137639] [GitHub Education] [answered]
### Hi everyone,

when I accept a github classroom assignment, I receive the following error:
![image](https://github.com/user-attachments/assets/83bbab39-70e2-4844-87db-db70b3dcf805)
After refreshing following the original link to accept the assignment I see the following:
![image](https://github.com/user-attachments/assets/4dd71e6d-8355-4acf-a8e4-caf64b5bc6c2)
After waiting for about 15 minutes I see this message:
![image](https://github.com/user-attachments/assets/0aacc77a-d3ad-4595-a841-d1475ef6cf1d)
When I reaccept the assignment I loop back through the same errors

Edit: I have attempted to accept the assignment in chrome, edge, and firefox

Edit: No longer an issue

**Comment by egrace479** [comment]
A bunch of my students are also having this error, while most have managed to accept and even submit their first assignment.

On my end I see repositories created for my students, but they're not connected to the classroom and the students don't have access. I also tried manually adding one student to his repo (giving him write access). He submitted the assignment and was autograded, but it still won't show up in the classroom.

**Comment by benjamingadd** [comment, marked as answer]
This issue is solved
